### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by Lino09**
## Project Setup

In this branch, I created the react app and set up the linters

### CODE REVIEW

File: src/reportWebVitals.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const reportWebVitals = (onPerfEntry) => {
```
Comment (GzimAsani) [inline src/reportWebVitals.js:1]: - [x] Unfortunately, I cannot run your app it is throwing me this error. KIndly fix it:
Screenshot from 2021-11-22 23-14-45

File: .stylelintrc.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["stylelint-config-standard"],
+  "plugins": ["stylelint-scss", "stylelint-csstree-validator"],
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "csstree/validator": true
+  },
+  "ignoreFiles": ["build/**", "dist/**", "**/reset*.css", "**/bootstrap*.css", "**/*.js", "**/*.jsx"]
+}
```
Comment (GzimAsani) [inline .stylelintrc.json:10]: - [x] The picture in the readme file is broken, kindly fix it. Also please be more specific on how the user can clone and run your app.

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @Lino09                              
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Great job with the project :muscle: 

## Required Changes ♻️ 
- [x] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/reportWebVitals.js (hunk 1 above)
Comment (Lino09) [reply src/reportWebVitals.js:1]: Hey @GzimAsani  thanks for your comment, do you think you could give me some guidance here? I just created the project And didn't even write any code, I cannot find a way to move that import to top-level without breaking the app, do you have any suggestions?

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Lino09, 

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**